noise_corrector.py

```diff
@@ -1,66 +1,211 @@
 #!/usr/bin/env python3
 
+import argparse
 import collections
+from datetime import timedelta
 import json
-import math
 import numpy
```

Comment:
> the event-level mode takes that aggregate debiased data and produces synthetic event-level data based off of it
What do you think the main use case would be for these? Simulation?

Reply:
It's a good question that I don't know the full answer to. I think it could be used for some data processing that involves more than just one aggregate (e.g. generating data to run multiple aggregate queries off of so they are consistent), or to integrate with existing code that would take a lot of effort to port over to accept aggregate data.

This is something that we should continue to explore with @pasin30055. For now, because we don't have a firm use-case I am going to mark the mode as experimental.